### PR TITLE
Update SiteContentFeaturedPage.php

### PR DESCRIPTION
**Pull request by kshitijSharma2014**

### CODE REVIEW

**Comment by kshitijSharma2014** [comment]
@puneet0191  @javigomez  @gunjanpatel  
please review it

thanks :)

File: tests/system/webdriver/Pages/SiteContent/SiteContentFeaturedPage.php [hunk 1]
```diff
@@ -1,11 +1,6 @@
 <?php
 
 use SeleniumClient\By;
-use SeleniumClient\SelectElement;
-use SeleniumClient\WebDriver;
-use SeleniumClient\WebDriverWait;
-use SeleniumClient\DesiredCapabilities;
-use SeleniumClient\WebElement;
```
Comment (puneet0191) [inline tests/system/webdriver/Pages/SiteContent/SiteContentFeaturedPage.php:8]: Why have you removed this?

Comment (kshitijSharma2014) [inline tests/system/webdriver/Pages/SiteContent/SiteContentFeaturedPage.php:8]: My phpcs was saying that these were never used
but if you I will add them.

**Comment by kshitijSharma2014** [comment]
@puneet0191 
 I have added those lines.

**Comment by kshitijSharma2014** [comment]
added a new function to check for voting option

**Pull request merged**